containers/package-builder/build-concent-api-assets.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+concent_dir=$1
+output_dir=$2
+
+mkdir --parents build/concent_api_assets/
+cp --recursive $concent_dir/concent_api build/concent_api_assets
+
+cd build/concent_api_assets/concent_api/
+
+virtualenv --python python3 concent-api-virtualenv/
+source concent-api-virtualenv/bin/activate
+pip install -r "requirements.txt"
```

Comment:
This fails on my machine:

```
package-builder/build-concent-api-assets.sh: /home/cameel/data/working-set/projects/golem/concent-deployment/containers/build/concent_api_assets/concent_api/concent-api-virtualenv/bin/pip: /home/cameel/data/working-set/projects/golem/concent-deployment/containers/build/concent_api_assets/concent_api/concent-api-v: bad interpreter: No such file or directory
make: *** [Makefile:44: build/nginx-proxy/concent-api-assets.tar] Error 126
```

Turns out that the length of the shebang line is limited to 127 characters in Linux (see [The #! magic, details about the shebang/hash-bang mechanism on various Unix flavours > Further history and the maximum length of a #! line](https://www.in-ulm.de/~mascheck/various/shebang/#length)). `virtualenv` embeds the full path to the python executable in all scripts in the `/bin/` directory, including `pip`. If you open the `pip` script, the first line looks like this:

``` python
#!/home/cameel/data/working-set/projects/golem/concent-deployment/containers/build/concent_api_assets/concent_api/concent-api-virtualenv/bin/python3
```

This is too long and gets cut off.

There are various workarounds but in our case the simplest one is to simply put the virtualenv in the `build/` directory rather than deeper, inside `concent-api-virtualenv/`.